app/controllers/chargeback_controller.rb

```diff
@@ -223,8 +221,7 @@ def cb_rates_delete
       javascript_flash
     else
       cb_rates_list
-      @right_cell_text = _("%{typ} %{model}") % {:typ   => x_node.split('-').last,
-                                                 :model => ui_lookup(:models => 'ChargebackRate')}
+      @right_cell_text = _("%{typ} Chargeback Rate") % {:typ => x_node.split('-').last}
```

Comment:
This looks strange, I mean the `:typ` part. I know it's not part of this PR, but it's an i18m issue I believe.